internal/featuresupport/support_level_feature.go

```diff
@@ -5,6 +5,12 @@ import (
 	"github.com/openshift/assisted-service/models"
 )
 
+// Interface for support level features that can validate themselves
+type ValidatingSupportLevelFeature interface {
```

Comment:
```suggestion
type SupportLevelFeatureValidator interface {
```
I think it would be more descriptive for an interface.